WordPress/Classes/System/WordPressAppDelegate+openURL.swift

```diff
@@ -29,7 +30,7 @@ import AutomatticTracks
             return JetpackNotificationMigrationService.shared.handleNotificationMigrationOnWordPress()
         }
 
-        guard url.scheme == WPComScheme else {
+        guard url.scheme == BuildSettings.current.appURLScheme else {
```

Comment:
Worth using `appURLScheme` to have a centralize access to `BuildSettings`?

```suggestion
        guard url.scheme == WordPressAppDelegate.appURLScheme else {
```

Or, given the hope to remove Objective-C code, it'd be best to not relay on that `@objc` code?

Reply:
> Or, given the hope to remove Objective-C code, it'd be best to not relay on that @objc code?

I added it temporarily as a way to use Swift-only code from Objective-C. The idea is to remove `WordPressAppDelegate.appURLScheme`. I'll add a note.

Reply:
👍